web/less/variables.less

```diff
@@ -103,3 +103,15 @@
 @smartphones: ~"only screen and (max-width: 480px), only screen and (-webkit-min-device-pixel-ratio: 1.5, -o-min-device-pixel-ratio: 3/2, min--moz-device-pixel-ratio: 1.5, min-device-pixel-ratio: 1.5)";
 @smartphonesLandscape: ~"@{smartphones} only screen and (orientation:landscape) and (max-width: 480px), only screen and (orientation:landscape) and (-webkit-min-device-pixel-ratio: 1.5, -o-min-device-pixel-ratio: 3/2, min--moz-device-pixel-ratio: 1.5, min-device-pixel-ratio: 1.5)";
 @smartphonesPortrait: ~"@{smartphones} only screen and (orientation:portrait) and (max-width: 480px), only screen and (orientation:portrait) and (-webkit-min-device-pixel-ratio: 1.5, -o-min-device-pixel-ratio: 3/2, min--moz-device-pixel-ratio: 1.5, min-device-pixel-ratio: 1.5)";
+
+@iphoneX: ~"only screen and (min-device-width : 375px) and (max-device-width : 812px) and (-webkit-device-pixel-ratio : 3)";
+@iphone678: ~"@media only screen and (min-device-width : 375px) and (max-device-width : 667px)";
+@iphone678Plus: ~"@media only screen and (min-device-width : 414px) and (max-device-width : 736px)";
+@iphone5n5Plus: ~"@media only screen and (min-device-width : 320px) and (max-device-width : 568px)";
+@iphone234: ~"@media only screen and (min-device-width : 320px) and (max-device-width : 480px)";
```

Comment:
@Raulkg Can we remove these lines? They are unused, right?

Reply:
yes but they help us reference all different devices so that we quickly reference if all the devices are served and if needed we can use them  for custom styling